pkg/util/maven/maven_command.go

```diff
@@ -74,6 +74,17 @@ func (c *Command) Do(ctx context.Context) error {
 		args = append(args, "--settings", settingsPath)
 	}
 
+	if GOMAXPROCS, ok := os.LookupEnv("GOMAXPROCS"); ok {
+		args = append(args, "-Dmaven.artifact.threads="+GOMAXPROCS)
+	}
+
+	if cpuQuota, ok := os.LookupEnv("CPU_QUOTA"); ok {
+		if !strings.Contains(c.context.AdditionalArguments[len(c.context.AdditionalArguments)-1], "-T") {
+			args = append(args, "-T"+cpuQuota)
```

Comment:
I think the idea here would be to set the `-T` option based on the `GOMAXPROCS` environment variable, as for the `maven.artifact.threads` option.